lib/consensus/isaac.go

```diff
@@ -167,6 +167,10 @@ func (is *ISAAC) StartSync(height uint64, nodeAddrs []string) {
 	return
 }
 
+// GetSyncInfo gets the height it needs to sync.
+// It returns height, node list and error.
+// The height is the smallest height above the threshold.
```

Comment:
Is it ? I thought it was the highest available height, looking at the code

Reply:
@Geod24 Exactly. We are on the same page, but my english aws awkward :) Thanks.